docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
   test-runner:
-    image: ruby:2.5
+    image: ruby:2.4
```

Comment:
Is this a deliberate downgrade?

Reply:
Yes - the CI build uses 2.4, so this has been changed to match that.

Reply:
👍